need default zone for provider

```
╷
│ Error: Cannot determine zone: set in this resource, or set provider-level zone.
│ 
│   with google_compute_instance.nginx_instance,
│   on nginx_proxy.tf line 5, in resource "google_compute_instance" "nginx_instance":
│    5: resource "google_compute_instance" "nginx_instance" {
│ 
╵
Error: Terraform exited with code 1.
Error: Process completed with exit code 1.
```
zoneが必要なのでdefault値を復活する
bugfix for #12

--- a/cicd/backup/state-bucket.tf
+++ b/cicd/backup/state-bucket.tf
@@ -1,7 +1,7 @@
 resource "google_storage_bucket" "terraform-state-store" {
   name          = "marltake-practice-cicd-2"
   project       = "practice-cicd-mtk"
-  location      = var.region
+  location      = "us-central1"
   storage_class = "REGIONAL"
 
   versioning {

--- a/cicd/main.tf
+++ b/cicd/main.tf
@@ -9,11 +9,8 @@ terraform {
 
 provider "google" {
   project = "practice-cicd-mtk"
-  region  = var.region
-}
-
-variable "region" {
-  default = "us-central1"
+  region  = "us-central1"
+  zone    = "us-central1-c"
 }
 
 resource "google_project_service" "service" {

